Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -769,23 +769,23 @@
 #define ENABLE_VORBIS 1
 #endif
 
-#if !defined(ENABLE_SET_WEBCONTENT_PROCESS_INFORMATION_IN_NETWORK_PROCESS) && PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000
+#if !defined(ENABLE_SET_WEBCONTENT_PROCESS_INFORMATION_IN_NETWORK_PROCESS) && PLATFORM(MAC)
 #define ENABLE_SET_WEBCONTENT_PROCESS_INFORMATION_IN_NETWORK_PROCESS 1
 #endif
 
 #if !defined(ENABLE_WEBM_FORMAT_READER) && PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 110300
 #define ENABLE_WEBM_FORMAT_READER 1
 #endif
 
-#if !defined(ENABLE_OPUS) && (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000 || PLATFORM(IOS) || PLATFORM(IOS_SIMULATOR) || PLATFORM(VISION))
+#if !defined(ENABLE_OPUS) && (PLATFORM(MAC) || PLATFORM(IOS) || PLATFORM(IOS_SIMULATOR) || PLATFORM(VISION))
```

Comment:
Suspicious about lack of tvOS and macCatalyst. Also, `IOS_SIMULATOR` is superfluous IIRC.

Reply:
Right, I think you're right that the original author probably made a mistake here, but I'd like these cleanup patches not to have behavior changes. I think after I'm done with this cleanup project I should audit all the Platform*.h files and try to find all situations like this and see if they can be adjusted.

Reply:
My implication was that a FIXME would be nice, as you added elsewhere.